packages/forma-36-website/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "@typescript-eslint/no-explicit-any": "off"
```

Comment:
We need to discover what is happening with this thing... I've just pulled your branch and removed this file so I this rule on basically and it worked! hahahha

I say that also because this rule is good... it forbids us from creating types like  `interface Props { prop: any }`